service/src/main/java/bio/terra/workspace/service/resource/referenced/cloud/gcp/ReferencedResourceService.java

```diff
@@ -139,10 +150,12 @@ public void updateReferenceResource(
               .resourceType(resource.getResourceType())
               .resourceName(name)
               .addParameter(ResourceKeys.RESOURCE_DESCRIPTION, description);
```

Comment:
Added cloning instructions here too. Unit test for this case?

Reply:
This method is tested in `bio.terra.workspace.service.resource.referenced.ReferencedResourceServiceTest#updateNameDescriptionAndCloningInstructions`.